cartaya/card-2/resources/css/style.css

```diff
@@ -1,11 +1,90 @@
-.card-group_container {
-    
+@font-face {
+    font-family: 'Camphor-Light';
+    src: url('./../font/Camphor-W01-Light.eot?#iefix') format('embedded-opentype'),
+         url('./../font/Camphor-W01-Light.woff') format('woff'),
+         url('./../font/Camphor-W01-Light.ttf') format('truetype'),
+         url('./../font/CamphorW01-Light.svg#FontName') format('svg');
+}
+
+body {
+    margin: 0;
+}
+.card-group {
+    align-items: center;
+    background-color: #8898aa;
+    background-color: rgba(136, 152, 170, 0.1);
+    display: flex;
+    height: 100vh;
+    justify-content: center;
 }
 
 .card {
-    width: 300px;
-	height: 226px;
-	border-radius: 4px;
-	background-color: #2196f3;
-	box-shadow: 0 2px 8px 0 rgba(0, 0, 0, 0.38);
-}
\ No newline at end of file
+    border-radius: 4px;
+    box-shadow: 0 2px 8px 0 rgb(90, 90, 90);
+    box-shadow: 0 2px 8px 0 rgba(0, 0, 0, 0.38);
+    min-height: 226px;
+    min-width: 300px;
+}
+
+.card_one {
+    background-color: #2196f3;
+    margin-right: 30px;
```

Comment:
You don't need apply this property to each element, you should be use a selector